FAQ.md

```diff
@@ -10,26 +10,43 @@ Feel free to add new questions or send comments to this page as pull requests._
 >- Or a single/few static IPs?
 >- Will one node be the designated WAN egress/ingress point or will there be multiple gateways on the meshnet?
 
+@jonah-archive is working with the venue and local ISPs to pool together some internets. I am imagining the network to primarily operate without a WAN connection, but expect one or more more nodes as gateway depending on how the connectivity is sourced. If there is a line that pops out somewhere vs. two providers where one is satellite, we could have multiple gateways.
+
+I don't expect static IPs would be needed as I don't expect to be hosting services to the outside world. Of course, if people run some distributed service with Internet peers, that is still possible.
+
 >Since the location is coastal (e.g. wind, moisture, dense fog, animals, etc.), what are the allowances for HA - high availability - radios, routers/switches
 >- You can use simple STP for fail-over between radios
```

Comment:
Wouldn't this be automatic with a mesh protocol like Babel or batman-adv?

Reply:
Yes, gateway selection and availability should be automatically announced/advertised with one of those routing protocols.